HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Resources/GiniColors.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  GiniColors.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import UIKit
+
+extension UIColor {
+    struct GiniColors {
+        static let accent1 = UIColorPreferred(named: "Accent01")
```

Comment:
[Request] For the Health SDK the color palette is different. Can you please update the colors in `GiniColors` struct and in the assets to match `Accent01`, `Accent02`, ..., `Dark01`, `Dark02`, ..., `Light01`, `Light02`, ..., `Feedback01`, `Feedback02`, ... in [this palette](https://www.figma.com/file/BMUKl0uCv6LyIW4kOAXCuD/Insurance-Mockup-%2F-Gini?type=design&node-id=4227-3821&mode=design&t=e7IjkgDJ9AGNPEJ8-0)? The green colors in the palette don't have a name so you can ignore them. Also you can remove the `Error..`, `Success..` and `Warning..` colors. Thank you!